operator/mysqlcontroller/observe.go

```diff
@@ -0,0 +1,180 @@
+package mysqlcontroller
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	exoscaleapi "github.com/exoscale/egoscale/v2/api"
+	"github.com/exoscale/egoscale/v2/oapi"
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
```

Comment:
```suggestion
	"fmt"
	"errors"
	"net/url"
	"strings"

	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
	"github.com/crossplane/crossplane-runtime/pkg/resource"
	exoscaleapi "github.com/exoscale/egoscale/v2/api"
	"github.com/exoscale/egoscale/v2/oapi"
	"github.com/go-logr/logr"
```

wrong errors package

Reply:
good catch